Source/MonoGame.Extended/Maps/Tiled/TiledTileLayer.cs

```diff
@@ -54,22 +54,59 @@ public override void Draw(Camera2D camera)
             {
                 var region = _map.GetTileRegion(tile.Id);
 
-                if (region != null)
-                {
-                    // not exactly sure why we need to compensate 1 pixel here. Could be a bug in MonoGame?
-                    var tx = tile.X * (_map.TileWidth - 1);
-                    var ty = tile.Y * (_map.TileHeight - 1);
-                        
-                    _spriteBatch.Draw(region, new Rectangle(tx, ty, region.Width, region.Height), Color.White);
-                }
+                if(region == null) continue;
```

Comment:
I agree that this is a good way to reduce nesting. Personally I find the code reads better paying the nesting cost though.